eventtracking/django/admin.py

```diff
@@ -0,0 +1,26 @@
+"""
+Contains Admin class(es) for the django app
+"""
+from django.contrib import admin
+
+from eventtracking.django.models import RegExFilter
+
+
+class RegExFilterAdmin(admin.ModelAdmin):
+    """
+    Admin model class for RegExFilter model.
+    """
+
+    def has_add_permission(self, request):
```

Comment:
This was added here to avoid showing the "Add button" once there was a filter created. But with the change of adding backend_name field, we don't require this anymore